src/shared/parameter_v2/scripts/type_map.py

```diff
@@ -0,0 +1,17 @@
+CPP_TYPE_MAP = {
+    "bool": "bool",
+    "int": "int",
+    "uint": "unsigned",
+    "float": "double",
+    "string": "std::string",
+    "factory": "std::string",
+    "enum": None,
```

Comment:
kinda out of scope, but shouldn't `enum` map to `enum`?

Reply:
it doesn't here because we typdef, so it would get its type during "generation time"